test/src/org/exist/xquery/XPathQueryTest.java

```diff
@@ -150,7 +150,17 @@
         "    <elem3>val2</elem3>\n" +
         " </elem2>\n" +
         "</elem1>";
-    
+
+    private final static String COLLECTION_CONF = "<collection xmlns=\"http://exist-db.org/collection-config/1.0\">" +
+            "    <index>\n" +
+            "        <!-- Range index for fn:id and fn:idref -->" +
```

Comment:
Ah, I see - I didn't know about `fn:idref()`.  Support for this is new to the new range index, then, right?  New to eXist too?